src/Tests/Entities/ProjectTest.php

```diff
@@ -4,6 +4,8 @@
 
 use DateInterval;
 use DateTime;
+use ProjectManagementApi\Exceptions\InvalidUserArrayDatatypeException;
+use ProjectManagementApi\Entities\User;
 use ProjectManagementApi\Entities\Project;
 use PHPUnit\Framework\TestCase;
 require '../../../vendor/autoload.php';
```

Comment:
this needs to be `require 'vendor\autoload.php` for our unit test to run with the other ones from the main folder.